packages/ckeditor5-list/src/list/utils.js

```diff
@@ -395,7 +395,6 @@ export function getSelectedListItems( model ) {
 		.filter( element => element.is( 'element', 'listItem' ) )
 		.map( element => {
 			const position = model.change( writer => writer.createPositionAt( element, 0 ) );
-
```

Comment:
Accidentally removed line break. Maybe something is wrong with your IDE settings?